ui-common/src/components/TrafficViewer/TrafficViewer.tsx

```diff
@@ -191,24 +188,24 @@ export const TrafficViewer: React.FC<TrafficViewerProps> = ({
 
   const toggleConnection = () => {
     if (!closeWebSocket()) {
-      openEmptyWebSocket();
-      scrollableRef.current.jumpToBottom();
-      setIsSnappedToBottom(true);
+      setIsStreamData(true)
```

Comment:
pls add a comment to describe that we have to wait for the fetch to finish before open the web socket